docs/dom-testing-library/api-events.mdx

```diff
@@ -96,6 +96,8 @@ fireEvent.keyDown(domNode, {key: 'Enter', code: 'Enter', charCode: 13})
 fireEvent.keyDown(domNode, {key: 'A', code: 'KeyA'})
 ```
 
+> :warning: `React` doesn't dispatch a `SyntheticEvent` for a native `keypress` event without `event.charCode === 13 || event.charCode >= 32`.
```

Comment:
If I'm not mistaken, this would be the first time that we mention something React specific into the core docs. From my point of view, I think we should keep the core docs framework agnostic, or else we can add more and more cases like these into the docs, which would make them harder to read.
What if we move this to the React FAQ or [cheatsheet](https://testing-library.com/docs/react-testing-library/cheatsheet#events)?
Thoughts @MatanBobi ?

Reply:
From the *resolved* conversation above:
> Note, that this is framework-specific and almost all use cases for this should use `user-event` instead.

Reply:
Yea, you're correct @ph-fritsche , but then maybe this comment shouldn't be added to the docs?

Reply:
Yes, this is some framework-specific knowledge that should be irrelevant to almost all developers.
The longer I think about it, the more I'm convinced that not adding such information would be better and that the fact that someone ran into this rather means we should do more to encourage people to use the right tool.
See https://github.com/testing-library/testing-library-docs/issues/963#issuecomment-1023150615